spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper' 
+
+describe PasswordResetsController do 
+
+  describe "GET show" do 
+
+    before do 
+      @alice = Fabricate(:user, token: '12345')
+    end
+
+    context "valid token" do 
+      it "renders the show template" do
+        @alice.update_column(:token, '12345') 
+        get :show, id: '12345'
+        expect(response).to render_template :show
+      end
+    end
+
+    context "invalid token" do 
+      it "redirects to expired token page" do 
+        get :show, id: ''
+        expect(response).to redirect_to expired_token_path
+      end
+    end
+  end
+
+  describe "POST create" do 
+
+    before do 
+      @alice = Fabricate(:user, token: '12345')
```

Comment:
You should use `let` rather than instance variables. Read more [here](http://betterspecs.org/#let).